functions/src/mappers/github/webhook-event-response/pull-request-review-comment.ts

```diff
@@ -0,0 +1,80 @@
+import { DocumentData } from '../../../client/firebase-admin';
+
+// Dashboard mappers/models
+import { GitHubPullRequestModel } from '../pullRequest.mapper';
+import { Logger } from './../../../client/logger';
+import { PullRequest } from './pull-request';
+import { isExistProperties, Repository, User } from './shared';
+
+interface Comment {
```

Comment:
Is this the `input`? Should it be named `CommentInput`?

Reply:
This is interface not Input.

Reply:
ok thanks 👍

Reply:
Reviewing this again, I think it is part of the `Input` interface on line 

```typescript
export interface PullRequestReviewCommentEventInput {
   action: Action;
   comment: Comment; // <-- here
   pull_request: PullRequest;
   repository: Repository;
   sender: User;
 }
```

https://github.com/DashboardHub/PipelineDashboard/pull/1591/files#diff-3774103d4dc00ec575def8a28b696cf6R25